packages/editor/src/pages/Editor/store/settings/sagas.ts

```diff
@@ -44,7 +48,12 @@ export const verifySettings = parsed => {
       !allowedSettings[setting].includes(parsed[setting])
     ) {
       /* In this case, there was a setting in parsed that wasn't in the list of allowed */
-      throw new Error(`'${parsed[setting]}' is not an allowed value for '${setting}'.`);
+      throw new Error(
+        `'${parsed[setting]}' is not an allowed value for '${setting}'. ` +
+          `Allowed values include: ${allowedSettings[setting]
```

Comment:
👍 Seems more helpful. How does it look inside a message bar though?

Reply:
@nico-bellante , it looks nice enough:

![image](https://user-images.githubusercontent.com/2230453/57049365-a559d700-6c2c-11e9-9dfc-74b168a1f939.png)